### ISSUE TITLE
npm install error with 5.0.6

### ISSUE THREAD

**Issue by dxdc** [techyowl/homebridge-august-smart-locks#24]
`sudo npm -g install homebridge-august-smart-locks@5.0.6`

Some issue with `august-connect` repo?

```
npm ERR! code 128
npm ERR! Command failed: git clone --mirror -q git://github.com/techyowl/august-connect.git /root/.npm/_cacache/tmp/git-clone-d31d5b05/.git
npm ERR! fatal: could not create leading directories of '/root/.npm/_cacache/tmp/git-clone-d31d5b05/.git'
npm ERR! 
```

**Comment by techyowl** [comment]
I'd purge the node_modules folder and npm install again. Seems to be an error with your npm cache as not everyone is having this issue. Also curious why you are using sudo with an npm install command that's dangerous. You may need to uninstall august-connect with npm remove then try installing the home bridge plugin again. You can also clone this repo run npm install in its directory and run npm link to add it to homebridge. There are a lot of ways to install it.

**Comment by dxdc** [comment]
I was able to reproduce the issue like this:

```sh
sudo su root
mkdir test && cd test
npm install github:techyowl/august-connect
```

I think the primary issue is here:

- You are referencing a github dependency in your `package.json`. It may be much cleaner to reference the ZIP file from a release. The problem is that it requires a user to have SSH access (meaning, they have already associated their machine's SSH key with their Github account). Referencing a ZIP doesn't have this requirement.

I was able to install the package a different way, but I think it may be better to replace the github dependency with just a 'release' ZIP from your repo and remove the requirement that a user has SSH access to their github account.
